testing/environments/local.yml

```diff
@@ -2,27 +2,31 @@
 # This is useful for testing locally with a full elastic stack setup.
 # All services can be reached through localhost like localhost:5601 for Kibana
 # This is not used for CI as otherwise ports conflicts could happen.
-version: '2'
+version: '2.1'
 services:
   kibana:
     ports:
-      - "5601:5601"
-    environment:
-      - ELASTICSEARCH_URL=http://elasticsearch:9200/
+      - "127.0.0.1:5601:5601"
 
   elasticsearch:
     ports:
-      - "9200:9200"
```

Comment:
It prevents docker from binding to 0.0.0.0 for the forwarded ports. And this prevents someone from connecting remotely to your Dockerized services. This will be good for security 🔒 .